interface/billing/sl_eob_search.php

```diff
@@ -186,74 +232,75 @@ function upload_file_to_client_pdf($file_to_send) {
+       $stmt['today'] = $today;
+       $stmt['duedate'] = $duedate;
+     } else {
         // Report the oldest due date.
-        if ($duedate < $stmt['duedate']) {
-          $stmt['duedate'] = $duedate;
-        }
+      if ($duedate < $stmt['duedate']) {
+        $stmt['duedate'] = $duedate;
       }
+    }
 
       // Recompute age at each invoice.
-      $stmt['age'] = round((strtotime($today) - strtotime($stmt['duedate'])) /
-        (24 * 60 * 60));
-
-      $invlines = ar_get_invoice_summary($row['pid'], $row['encounter'], true);
-      foreach ($invlines as $key => $value) {
-        $line = array();
-        $line['dos']     = $svcdate;
-        if ($GLOBALS['use_custom_statement']) {
-	      $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : $value['code_text'];
-		}
-        else
-		{
-        $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : "Procedure $key";
-	    }
-        $line['amount']  = sprintf("%.2f", $value['chg']);
-        $line['adjust']  = sprintf("%.2f", $value['adj']);
-        $line['paid']    = sprintf("%.2f", $value['chg'] - $value['bal']);
-        $line['notice']  = $duncount + 1;
-        $line['detail']  = $value['dtl'];
-        $stmt['lines'][] = $line;
-        $stmt['amount']  = sprintf("%.2f", $stmt['amount'] + $value['bal']);
-		$stmt['ins_paid']  = $stmt['ins_paid'] + $value['ins'];
-      }
+    $stmt['age'] = round((strtotime($today) - strtotime($stmt['duedate'])) /
+      (24 * 60 * 60));
+
+    $invlines = ar_get_invoice_summary($row['pid'], $row['encounter'], true);
+    foreach ($invlines as $key => $value) {
+      $line = array();
+      $line['dos']     = $svcdate;
+      if ($GLOBALS['use_custom_statement']) {
+       $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : $value['code_text'];
+     } else {
+      $line['desc']    = ($key == 'CO-PAY') ? "Patient Payment" : "Procedure $key";
+    }
+    $line['amount']  = sprintf("%.2f", $value['chg']);
+    $line['adjust']  = sprintf("%.2f", $value['adj']);
+    $line['paid']    = sprintf("%.2f", $value['chg'] - $value['bal']);
+    $line['notice']  = $duncount + 1;
+    $line['detail']  = $value['dtl'];
+    $stmt['lines'][] = $line;
+    $stmt['amount']  = sprintf("%.2f", $stmt['amount'] + $value['bal']);
+    $stmt['ins_paid']  = $stmt['ins_paid'] + $value['ins'];
+  }
 
       // Record that this statement was run.
-      if (! $DEBUG && ! $_POST['form_without']) {
-        sqlStatement("UPDATE form_encounter SET " .
-          "last_stmt_date = '$today', stmt_count = stmt_count + 1 " .
-          "WHERE id = " . $row['id']);
-      }
-    } // end for
+  if (! $DEBUG && ! $_POST['form_without']) {
+    sqlStatement("UPDATE form_encounter SET " .
+      "last_stmt_date = '$today', stmt_count = stmt_count + 1 " .
+      "WHERE id = " . $row['id']);
+  }
+    } // end while
 
     if (!empty($stmt)) ++$stmt_count;
-    fwrite($fhprint, create_statement($stmt));
+  //  fwrite($fhprint, create_HTML_statement($stmt));
```

Comment:
The fwrite($fhprint, create_statement($stmt)) line was listed twice in the original file.  
Perhaps Rod knows why, first prior to making the $stmt array (64 lines up) then here...  
We only need one...